### PR TITLE
Update instructions to echo to Cloud loaded webpage

### PR DESCRIPTION
**Pull request by kieranacquia**
In the UX session we saw that the user should be able to load the ode and see validating content.

### CODE REVIEW

File: tutorial-701/acquia-pipelines.yaml [hunk 1]
```diff
@@ -15,7 +15,8 @@ events:
       # If you have build instructions, put them here. They are not required.
       - build_site:
           script:
-            - echo "Put your build instructions, if any, here."
+            - touch index.html
+            - echo "Pipelines examples 701. You successfully deployed your build to an ODE" > index.html
```
Comment (unn) [inline tutorial-701/acquia-pipelines.yaml]: If you want this to work, you should remove the touch, and echo to docroot/index.html

**Review by unn — CHANGES_REQUESTED** [review]

**Comment by unn** [comment]
@kieranacquia Please update your pull request based on the feedback.

**Comment by bjaspan** [comment]
Duplicates #15, already committed.

**Comment by bjaspan** [comment]
Oops, #15 merged into this (#13). Now merging this to master.

**Pull request merged**